utils/build.ps1

```diff
@@ -3207,6 +3207,10 @@ if (-not $SkipBuild) {
 
     # Build platform: SDK, Redist and XCTest
     Invoke-BuildStep Build-Runtime Windows $Arch
+
+    # FIXME(#79839)
+    Get-ChildItem -Path "$($Arch.SDKRoot)\usr\lib\swift\android" -Filter "*.swiftinterface" -Recurse
```

Comment:
Was this supposed to look into `\usr\lib\swift\windows`? I think we can skip it altogether as the Windows SDKs don't reproduce the error.

Reply:
Yes, it was supposed to look into Windows. I've tested it locally and definitely see the issues on Windows locally :)

Reply:
Interesting, doesn't happen for me locally